### ISSUE TITLE
DISABLED test_stress_heavy_rpc_torchscript (__main__.RpcTestWithSpawn)

### ISSUE THREAD

**Issue by mrshenli** [pytorch/pytorch#34237]
`test_stress_heavy_rpc_torchscript` is flaky

https://app.circleci.com/jobs/github/pytorch/pytorch/4688332

```
Mar 04 09:25:35 ======================================================================
Mar 04 09:25:35 ERROR [361.213s]: test_stress_heavy_rpc_torchscript (__main__.RpcTestWithSpawn)
Mar 04 09:25:35 ----------------------------------------------------------------------
Mar 04 09:25:35 Traceback (most recent call last):
Mar 04 09:25:35   File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/testing/_internal/common_distributed.py", line 175, in wrapper
Mar 04 09:25:35     self._join_processes(fn)
Mar 04 09:25:35   File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/testing/_internal/common_distributed.py", line 256, in _join_processes
Mar 04 09:25:35     self._check_return_codes(elapsed_time)
Mar 04 09:25:35   File "/Users/distiller/workspace/miniconda3/lib/python3.7/site-packages/torch/testing/_internal/common_distributed.py", line 275, in _check_return_codes
Mar 04 09:25:35     raise RuntimeError('Process {} terminated or timed out after {} seconds'.format(i, elapsed_time))
Mar 04 09:25:35 RuntimeError: Process 1 terminated or timed out after 361.193176984787 seconds
```

cc @ezyang @gchanan @zou3519 @suo @pietern @mrshenli @pritamdamania87 @zhaojuanmao @satgera @gqchen @aazzolini @rohan-varma @xush6528 @jjlilley @osalpekar

**Comment by zhaojuanmao** [comment]
@mrshenli was it disabled? or should I disable it now?

**Comment by mrshenli** [comment]
> was it disabled? or should I disable it now?

@zhaojuanmao 

It should have already been disabled by our chrono job. Currently disabled tests can be found here: https://github.com/zdevito/pytorch_disabled_tests/blob/master/result.json

**Comment by zhaojuanmao** [comment]
GIL change in #33329 could cause deadlock

**Comment by mrshenli** [comment]
closed by #34505
